packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -1064,21 +1069,92 @@ class AnimatedBuilder extends AnimatedWidget {
        assert(builder != null),
        super(listenable: animation);
 
-  /// Called every time the animation changes value.
+  /// Called every time the animation passed to the constructor changes value.
   final TransitionBuilder builder;
 
   /// The child widget to pass to the [builder].
   ///
-  /// If a [builder] callback's return value contains a subtree that does not
-  /// depend on the animation, it's more efficient to build that subtree once
-  /// instead of rebuilding it on every animation tick.
+  /// {@macro flutter.widgets.transitions.AnimatedBuilder.optimizations}
+  final Widget? child;
+
+  @override
+  Widget build(BuildContext context) {
+    return builder(context, child);
+  }
+}
+
+/// A general-purpose widget for building a widget subtree when a [Listenable]
```

Comment:
nit for these docs: per style guide they should be using passive voice and avoid you/your.